gitlab/datadog_checks/gitlab/metrics.py

```diff
@@ -3,152 +3,211 @@
-    "geo_attachments_failed": "geo.attachments_failed",
-    "geo_last_event_id": "geo.last_event_id",
-    "geo_last_event_timestamp": "geo.last_event_timestamp",
-    "geo_cursor_last_event_id": "geo.cursor_last_event_id",
-    "geo_cursor_last_event_timestamp": "geo.cursor_last_event_timestamp",
-    "geo_status_failed_total": "geo_status_failed_total",
-    "geo_last_successful_status_check_timestamp": "geo.last_successful_status_check_timestamp",
-    "geo_lfs_objects_synced_missing_on_primary": "geo.lfs_objects_synced_missing_on_primary",
-    "geo_job_artifacts_synced_missing_on_primary": "geo.job_artifacts_synced_missing_on_primary",
-    "geo_attachments_synced_missing_on_primary": "geo.attachments_synced_missing_on_primary",
-    "geo_repositories_checksummed_count": "geo.repositories_checksummed_count",
-    "geo_repositories_checksum_failed_count": "geo.repositories_checksum_failed_count",
-    "geo_wikis_checksummed_count": "geo.wikis_checksummed_count",
-    "geo_wikis_checksum_failed_count": "geo.wikis_checksum_failed_count",
-    "geo_repositories_verified_count": "geo.repositories_verified_count",
-    "geo_repositories_verification_failed_count": "geo.repositories_verification_failed_count",
-    "geo_repositories_checksum_mismatch_count": "geo.repositories_checksum_mismatch_count",
-    "geo_wikis_verified_count": "geo.wikis_verified_count",
-    "geo_wikis_verification_failed_count": "geo.wikis_verification_failed_count",
-    "geo_wikis_checksum_mismatch_count": "geo.wikis_checksum_mismatch_count",
-    "geo_repositories_checked_count": "geo.repositories_checked_count",
-    "geo_repositories_checked_failed_count": "geo.repositories_checked_failed_count",
-    "geo_repositories_retrying_verification_count": "geo.repositories_retrying_verification_count",
-    "geo_wikis_retrying_verification_count": "geo.wikis_retrying_verification_count",
-    "db_load_balancing_hosts": "db_load_balancing_hosts",
-    "unicorn_active_connections": "unicorn.active_connections",
-    "unicorn_queued_connections": "unicorn.queued_connections",
-    "unicorn_workers": "unicorn.workers",
-    "puma_workers": "puma.workers",
-    "puma_running_workers": "puma.running_workers",
-    "puma_stale_workers": "puma.stale_workers",
-    "puma_running": "puma.running",
-    "puma_queued_connections": "puma.queued_connections",
-    "puma_active_connections": "puma.active_connections",
-    "puma_pool_capacity": "puma.pool_capacity",
-    "puma_max_threads": "puma.max_threads",
-    "puma_idle_threads": "puma.idle_threads",
-    "puma_killer_terminations_total": "puma.killer_terminations_total",
+    'action_cable_active_connections': 'action_cable.active_connections',
+    'auto_devops_pipelines_completed_total': 'auto_devops_pipelines_completed_total',
+    'db_load_balancing_hosts': 'db_load_balancing_hosts',
+    'db_partitions_missing': 'db_partitions_missing',
+    'db_partitions_present': 'db_partitions_present',
```

Comment:
(?)

```suggestion
    'db_load_balancing_hosts': 'db.load_balancing_hosts',
    'db_partitions_missing': 'db.partitions_missing',
    'db_partitions_present': 'db.partitions_present',
```

Reply:
`db_load_balancing_hosts` already exists so keeping the current format here instead of creating a new `db` namespace but I agree that would have worked better.